pilot/utils/files.py

```diff
@@ -18,12 +18,17 @@ def setup_workspace(args) -> str:
     Also creates a 'tests' folder inside the workspace.
     :param args: may contain 'root' key
     """
-    root = args.get('root') or get_parent_folder('pilot')
-    name = args.get('name', 'default_project_name')
-    project_path = create_directory(os.path.join(root, 'workspace'), name)
+    workspace = args.get('workspace')
```

Comment:
@LeonOstrez I've also restored this as it broke a unit test.

The `workspace` and `root` args were both introduced at the same time. The difference is that `workspace` can be used to select an existing project workspace, whereas `root` will create a new directory

`'{root}/workspace/{name}' == workspace`